scripts/dts/extract_dts_includes.py

```diff
@@ -61,16 +61,22 @@ def generate_prop_defines(node_path, prop):
     # Generates #defines (and .conf file values) from the prop
     # named 'prop' on the device tree node at 'node_path'
 
+    parent_path = get_parent_path(node_path)
     binding = get_binding(node_path)
     if 'parent' in binding and 'bus' in binding['parent']:
         # If the binding specifies a parent for the node, then include the
         # parent in the #define's generated for the properties
-        parent_path = get_parent_path(node_path)
         def_label = 'DT_' + get_node_label(parent_path) + '_' \
                           + get_node_label(node_path)
     else:
         def_label = 'DT_' + get_node_label(node_path)
 
+    if prop == 'label' and \
+       parent_path and 'label' in reduced[parent_path]['props']:
```

Comment:
`get_parent_path('/')` currently returns `'/'`, so the `parent_path` check will always succeed.

Maybe the check could be `node_path != '/'` instead, if it's needed.

Reply:
Returning `None` for `/` might be best. Just ran into another case where it's useful.

Reply:
Did that in https://github.com/zephyrproject-rtos/zephyr/pull/13922. Just added

```
    if node_path == '/':
        return None
```

to the start of `get_parent_path()`.

Reply:
> Did that in #13922.

Okay. So this PR just needs to wait until #13922 gets merged.
Actually, even with the current master everything will work. `parent_path` will always succeed, but for top-level nodes `reduced['/']` won't have the 'label' property.

Reply:
I have just took another look. Checking `parent_path` here doesn't make much sense, as it will never be `None`. This code is executed only for nodes that have the `compatible` property, hence for path `'/'` we won't get here. I think I'll remove this check.

Reply:
Yeah, no need to avoid merge conflicts with those cleanups PRs either.